pl_examples/hydra_examples/conf/pl/checkpoint/modelckpt.yaml

```diff
@@ -0,0 +1,12 @@
+# @package _group_
```

Comment:
Why do you need these @package comments? Sorry i'm not as familiar with hydra

Reply:
See [this])(https://hydra.cc/docs/next/upgrades/0.11_to_1.0/adding_a_package_directive) and [this](https://hydra.cc/docs/next/advanced/overriding_packages)